docs/recommendations/protein/frameshift.md

```diff
@@ -34,18 +34,19 @@ Format:   **"prefix""amino_acid"position"new_amino_acid""fs""Ter""position_termi
     * extending the amino acid sequence at the C-terminal end with one or more amino acids are described as [Extension](../extension/)
     * containing a **translation stop codon** in the inserted sequence are described as an insertion, not as a deletion-insertion removing the entire C-terminal amino acid sequence
     * encoding an open reading frame but after the DNA insertion shifting to another reading are described as a **frame shift**
+    
 ## Examples
 
 * p.Arg97ProfsTer23 (**short** p.Arg97fs) / p.Arg97Profs\*23: a variant with Arg97 as the first amino acid changed, shifting the reading frame, replacing it for a Pro and terminating at position Ter23.
-* p.(Tyr4\*): the predicted consequence at the protein level of the variant ATGGATGCATACGTCACG.. to ATGGATGCA<u>TA<font color="red">\_</font>GTCA</u>CG (c.12delC) is a Tyr to translation termination codon.
+* p.(Tyr4\*): the predicted consequence at the protein level of the variant ATGGATGCATACGTCACG... to ATGGATGCATA<span class="spotlight">\_</span>GTCACG (c.12delC) is a Tyr to translation termination codon.
```

Comment:
Yes, this is another example of docs that need human editing. The reason I wrote the style guide is to help us address cases like this. We now need to go implement it. I just committed a8e8743 to show where I intend all of this to go. It now looks like this:

![image](https://github.com/HGVSnomenclature/hgvs-nomenclature/assets/109453/6ef96c86-0844-49bd-b9fc-a6c322569495)

Reply:
That looks good, and more concise than the original. However, having some additional annotation that somehow highlights the `TA` before the deletion and the `G` after the deletion would clarify the creation of a stop codon. Since the deleted base is already (clearly!) highlighted by a color, I would recommend a style change other than color. We could still use underlines, or make the bases (semi-)bold. Although currently, only the background color is changed, a font color would still be a possibility. We should then, however, think about color-blind users.

Reply:
Please check out 63bd839. Looks like this now:
![image](https://github.com/HGVSnomenclature/hgvs-nomenclature/assets/109453/0a4cf75c-da7f-443d-8d76-cd87e077d048)

Reply:
Very very nice! There's one redundant `<code></code>` pair, but those are details. It looks great!

Reply:
@ifokkema Are you referring to the `<code> </code>` with a space? If so, that's actually required because the surrounding blocks are classed differently and I wanted a plain (gray) background space there.